tests/integration/test_patches.py

```diff
@@ -18,20 +33,24 @@ def git_repo(tmpdir):
     """
     repo = git.Repo.init(tmpdir)
     shutil.copytree(
-        src="tests/data/patches/previous/",
+        src=f"{TESTS_DIR}/data/patches/previous/",
         dst=repo.working_tree_dir,
         dirs_exist_ok=True,
     )
     repo.git.add(repo.working_tree_dir)
     repo.git.commit("-mInitial patches")
     shutil.copytree(
-        src="tests/data/patches/regenerated/",
+        src=f"{TESTS_DIR}/data/patches/regenerated/",
         dst=repo.working_tree_dir,
         dirs_exist_ok=True,
     )
     return repo
 
 
+@pytest.mark.skipif(
+    not check_copytree_dirs_exists_support(),
+    reason="Old python version does not support copytree exists dirs parameter",
+)
```

Comment:
Isn't this #1160 ?

Reply:
right it is exactly this issue, so it will be fixed by this PR